src/amber/controller/helpers/responders.cr

```diff
@@ -35,13 +45,14 @@ module Amber::Controller::Helpers
       end
```

Comment:
Can we move `@type ||=` into this method? rather than needing the begin statement down below.
```crystal
def type
  @type ||= select_type.to_s
end
```
I like the convention of 
```crystal
def property_name
  @property_name ||= calculate_property_name
end
```
Of course, 'calculate' can be 'select' or 'generate' - what ever verb makes sense.

Reply:
I would have done that but I needed to return @type.to_s so that @type wouldn't have the possibility of being Nil.

Reply:
I did come up with a solution here that I think we'll both like.